site/.storybook/main.js

```diff
@@ -11,4 +14,19 @@ module.exports = {
     name: "@storybook/react-vite",
     options: {},
   },
+  async viteFinal(config, { configType }) {
+    config.plugins = config.plugins || []
+    // return the customized config
+    if (configType === "PRODUCTION") {
+      // ignore @ts-ignore because it's not in the vite types yet
+      config.plugins.push(
+        turbosnap({
+          rootDir: config.root || "",
+        }),
+      )
+    }
+    return mergeConfig(config, {
+      // Your environment configuration here
+    })
```

Comment:
Nit: Can we just return the `config`?

Reply:
Good call, that should work.
Let me test the build locally to verify everything works as expected.

Reply:
@BrunoQuaresma 
I made the suggested changes.